### DISCUSSION TITLE
Comfy batch more than 100 ..?

### DISCUSSION THREAD

**Discussion by Khampol** [comfyanonymous/ComfyUI#5952] [Ideas]
Hello, I just notice that comfyui bath is limit 100. 
Question : Could you add more? Until 9999 ? ... :>

**Comment by LukeG89** [comment]
You can edit the limit in Settings

![image](https://github.com/user-attachments/assets/9ee29ffa-250b-4b40-b2ac-d4c2dfe4fc49)

**Comment by Khampol** [comment]
oh yes ! 
Thanks a lot 👍❤️
